hack/upload-tutorial-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+CURRENT_DIR=$(dirname $0)
+PROJECT_ROOT="${CURRENT_DIR}"/..
+
+if ! command -v landscaper-cli &> /dev/null
+then
+    echo "landscaper-cli could not be found"
+    echo "install it by running: make install-cli"
+    exit
+fi
+
+blueprints=(
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/ingress-nginx:v0.2.1,./docs/tutorials/resources/ingress-nginx/blueprint"
```

Comment:
should we make the version configurable?

Reply:
You mean configurable by a a argument oder through a variable?
 
Making it fully configurable is not that easily possible as other resources still need to be adjusted.
 in the component descriptor and .
e.g 
- the installation: https://github.com/gardener/landscaper/blob/master/docs/tutorials/resources/ingress-nginx/installation.yaml#L16
- the component descriptor: https://github.com/gardener/landscaper/blob/master/docs/tutorials/resources/ingress-nginx/component-descriptor.yaml#L28

Reply:
I guess making it configurable via an argument to the upload script would be a good first step.

Reply:
don't know if configuration through an argument makes sense.
We would have to add at least 2 vars for the nginx and the other blueprints.

I think there is no benefit for exposing these variables as you would have to edit some resources in the repo anyway.
If It hink it makes more sense to set these versions as variable at the beginning of the script to make it more readable.

Then we would have the versions aligned in one repo and someone that runs the script would upload the approriate versions.

Reply:
👍

Reply:
@hendrikKahl PTAL

Reply:
looks good, thanks!